item-spec/item-spec.md

```diff
@@ -252,6 +218,48 @@ in the file).
 For examples of fields that this construct is recommended for, see the [section of STAC Best Practices](../best-practices.md#common-use-cases-of-additional-fields-for-assets)
 that talks about common use cases of additional fields for assets.
 
+## Media Types
+
+One of the best ways to help inform web clients about the content in a link is to use a common [Media 
+Type](https://en.wikipedia.org/wiki/Media_type) in the `type` field. In STAC the `type` field is used in both the 
+`[Link](#link-object)` and `[Asset](#asset-object)` Objects. It is quite useful for STAC browsers to better determine
+what to render and display to users searching and browsing the catalog.  Media types are often referred to by the 
+now-deprecated term "MIME types". 
```

Comment:
Not sure, is that a typo?
```suggestion
now deprecated term "MIME types". 
```

Reply:
I don't think it's a typo because both hyphenated words are describing `term`, but I wasn't an English major 😄

Reply:
I'm not even a native speaker, just haven't seen it yet with a hypen so was confused.

Reply:
Well I was trying to say that even native speakers have a hard time knowing when to use hyphens correctly 😄

Reply:
I think I didn't write that originally (copied it from another part of the spec). It may technically be correct but I think it's weird, so let's go with a space. Stated in another way - if I actually wrote that it's a typo, as I personally wouldn't mean to write it that way, but perhaps some english major did ;)